### PR TITLE
Fix crash on Windows when starting from a console

### LINKED ISSUE #2877: Crash on startup when starting cutter from cmd.exe or wt.exe
**Environment information**
* Operating System: windows 10
* Cutter version: 2.0.4
* Obtained from:
  - [ ] Built from source
  - [x] Downloaded release from Cutter website or GitHub 
  - [ ] Distribution repository
* File format: 
 * Arch: N/A
 * Type: N/A

**Describe the bug**

Crash on startup when starting cutter from cmd.exe or wt.exe (Windows Terminal) & PowerShell 7.2.1 on Windows 10.0.19044.0.

**To Reproduce**

1. Start `cutter` from either `cmd` or `wt` without any additional arguments.

**Expected behavior**

It shouldn't crash...

By the way? What's up with the weird start cutter from console to redirect output prompt? It shouldn't really be necessary to start a program from a console to redirect stdin/stdout/stderr, you can reassign them even if not started from the console as far as I'm aware. Just note that Python and other code grabs them on startup, so they should be initialized early enough.

**Additional context**

[Cutter_crash_dump_12.01.22_19.54.53.zip](https://github.com/rizinorg/cutter/files/7856672/Cutter_crash_dump_12.01.22_19.54.53.zip)

### PR DESCRIPTION
**Pull request by segevfiner**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://cutter.re/docs/contributing/code/getting-started.html) to this repository
- [x] I made sure to follow the project's [coding style](https://cutter.re/docs/contributing/code/development-guidelines.html)
- [x] I've updated the [documentation](https://cutter.re/docs/user-docs.html) with the relevant information (if needed)

**Detailed description**

See https://github.com/rizinorg/cutter/issues/2877#issuecomment-1020275956

> You crash here https://github.com/rizinorg/cutter/blob/91d99ba2190f4796da268be376d9f06e3c7f31bc/src/widgets/ConsoleWidget.cpp#L454
> Because `fileno(stdin)` is `-2` as cutter is built with `/SUBSYSTEM:WINDOWS` and hasn't set `stdin` to anything after calling `AttachConsole`. Honestly, you should be able to redirect them even when cutter is not attached to a console by just setting them to `NUL` or something else appropriate if you want the `fdopen` for saving the orig handles to not fail, or even just acknowledge that there are no original handles.
>
> And of course, any such stdio init should be done before anything that grabs/uses stdio itself, such as Python init, or rizin itself.

**Test plan (required)**

Manually started cutter on Windows from a console, it doesn't crash after the change.

(This is so small, that I haven't bothered, but let me know if needed)

**Closing issues**

Closes #2877

### CODE REVIEW

**Review by xarkes — APPROVED** [review]
Didn't test, LGTM.

File: src/Main.cpp [hunk 1]
```diff
@@ -22,14 +22,19 @@ static void connectToConsole()
         return;
     }
 
-    // Avoid reconfiguring stderr/stdout if one of them is already connected to a stream.
+    // Avoid reconfiguring stdin/stderr/stdout if one of them is already connected to a stream.
     // This can happen when running with stdout/stderr redirected to a file.
-    if (0 > fileno(stdout)) {
-        // Overwrite FD 1 and 2 for the benefit of any code that uses the FDs
+	if (0 > fileno(stdin)) {
```
Comment (ret2libc) [inline src/Main.cpp]: why the additional indentation?

Comment (segevfiner) [reply src/Main.cpp]: Somehow a tab has snuck there...

**Review by yossizap — APPROVED** [review]
LGTM tested with Windows

**Comment by ret2libc** [comment]
Thanks!

**Pull request merged**